src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -285,23 +307,27 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
             />
           </Flex>
         </Flex>
-        {bnOrZero(currentCapFillPercentage).eq(100) ? (
+        {isHardCapReached || bnOrZero(currentCapFillPercentage).eq(100) ? (
           <Alert status='warning' flexDir='column' bg={alertBg} py={4}>
             <AlertIcon />
             <AlertTitle>{translate('defi.modals.saversVaults.haltedTitle')}</AlertTitle>
-            <AlertDescription>
-              {translate('defi.modals.saversVaults.haltedDescription')}
-            </AlertDescription>
-            <Button
-              as={Link}
-              href={`https://twitter.com/intent/tweet?text=Hey%20%40THORChain%20%23raisethecaps%20already%20so%20I%20can%20deposit%20%23${underlyingAsset?.symbol}%20into%20a%20savers%20vault%20at%20%40ShapeShift`}
-              isExternal
-              mt={4}
-              colorScheme='twitter'
-              rightIcon={<FaTwitter />}
-            >
-              @THORChain
-            </Button>
+            {!isHardCapReached && (
```

Comment:
We don't want to show the "bug THORChain on Twitter" description and button here - this isn't about raising the caps, this is a hollistic max. across the network